Cargo.toml

```
@@ -0,0 +1,35 @@
+[package]
+name = "Slushie"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
```

Let's refactor authors to
```suggestion
authors = ["4IRE <hp@4irelabs.com>"]
```

---

> Let's refactor authors to

Why HP?

---

it's the contact email we have on our website, isn't it?